### DISCUSSION TITLE
Résumé de la demande

### DISCUSSION THREAD

**Discussion by maximpouliot** [MTESSDev/FRW#111] [Annonces]
Nous avons fait un lab client avec notre premier formulaire. 5 utilisateurs ont essayés le service et les commentaires ont été très bons. 

Un besoin qui est ressorti est d'avoir un résumé de la demande après avoir validé avant de l'envoyer officiellement. 

Est-ce que ça serait envisageable dans les prochaines versions? 

Merci

**Comment by LarcM7** [comment]
Bonjour,

Est-ce qu'il serait possible de nous donner un petit peu plus de détails sur ce besoin? Décrire la façon dont vous voyez cette fonctionnalité? Décrire le résumé de la demande, de quelle façon il s'affiche, quelles informations il contient, toute autre information pertinente. Vos idées!

Merci d'avance.

**Reply by maximpouliot** [reply]
Allo! 

Une fois le formulaire validé, ce serait possible de voir tout ce que le client nous envoi (résumé) comme information avant de cliquer sur Soumettre? 

Il me semble que c'est le genre de fonction qu'on voit régulièrement dans les services en ligne

J'espère que je suis un peu plus clair! ;) 

Merci!

**Comment by LarcM7** [comment]
Bonjour, 
Merci pour le complément d'information. 
Pour la suite, je vous invite à ouvrir une issue si vous souhaitez officiellement faire la demande pour cette fonctionnalité.
Merci.
